{
    "problem_identifier": "next_permutation",
    "prompt_identifier": "31",
    "model_identifier": "gpt-4",
    "score": 0.0,
    "sub_criteria_scores": null,
    "issues": [
        "Test failed: Input: nums = [1,2,3]; Expected Output: [\"1,3,2\"] Result: [1, 3, 2]",
        "Test failed: Input: nums = ['[3,2,1]']; Expected Output: [\"1,2,3\"] Result: ['[3,2,1]']",
        "Test failed: Input: nums = [1,1,5]; Expected Output: [\"1,5,1\"] Result: [1, 5, 1]"
    ]
}